src/orqviz/hessians/hessians.py

```diff
@@ -63,8 +64,16 @@ def get_Hessian(
         eps: Finite difference stencil used for numerical gradient in calculation.
             It is always used, even if gradient function is provided. Defaults to 0.1.
     """
-
-    n_params = len(params)
+    # if len(np.shape(params)) > 1:
```

Comment:
I think you can delete this instead of uncommenting?

Reply:
e1d12e2726eb322a5c4f38f30f61facff135f637